scripts/devaccess/devaccess_wrap.py

```diff
@@ -270,19 +304,51 @@ def rpm_qa(cmd):
         return results
 
     @staticmethod
-    def oc_get_nodes(occmd):
-        ''' provide 'oc get nodes' '''
+    def cmd_with_tail(cmd):
+        ''' replace process with called one to keep getting updates '''
+        args = cmd.split()
+        os.execvp(args[0], args)
 
-        n_cmd = occmd.normalized_cmd()
-        run_cmd = WhitelistedCommands.oc_cmd_builder(n_cmd)
-        results = subprocess.check_output(run_cmd)
+    @staticmethod
+    def safe_command(cmd):
+        ''' return any safe command output
+            this is for all commands that do not have any type of sensitive data in them
```

Comment:
This line concerns me.  The commands may only not have sensitive data due to careful control of arguments.  It would be useful to have "will never have sensitive data", "should never have sensitive data" (e.g. "ps" listing that may contain a password if someone foolishly includes on the command line), and "we take care to avoid it having sensitive data" (e.g. "cat" where we strongly restrict valid args).